sky_alert/endpoint.py

```diff
@@ -7,6 +14,27 @@
 def healthz():
     return 'OK'
 
-@app.get('/sunrise')
-def sunrise():
-    return '06:00'
\ No newline at end of file
+@app.get('/sundata')
+def sun_data(lat=0, lon=0):
```

Comment:
what do you think about using type hints? 
```suggestion
def sun_data(lat: str = 0, lon: str = 0) -> SunData:
```

Reply:
Great idea! I think it takes integers so will make that change